README.md

```diff
@@ -687,36 +688,53 @@ npx turbo --help
 
 #### Commands
 
-##### `crypto-fund`
+##### `balance`
 
-Fund a wallet with Turbo Credits by submitting a payment transaction for the crypto amount to the Turbo wallet and then submitting that transaction id to Turbo Payment Service for top up processing.
+Get the balance of a wallet or native address in Turbo Credits.
 
 Command Options:
 
-- `-v, --value <value>` - Amount of tokens in the token type's smallest unit value to fund the wallet with
+- `-a, --address <address>` - Address to get the balance of
```

Comment:
Use either `normalized address` or `native address`. `Address` is now ambiguous.